pkg/render/intrusion_detection.go

```diff
@@ -29,6 +29,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	v3 "github.com/tigera/api/pkg/apis/projectcalico/v3"
+
```

Comment:
Looks like PSP has been removed from the operator code already for this file, what keeps us from removing it everywhere else?

Reply:
I can remove PSPs in another PR.

Reply:
PSPs are removed in https://github.com/tigera/operator/pull/3374.